app/types/framework.tsx

```diff
@@ -18,26 +18,14 @@ type WidgetType = 'excerptWidget'
     | 'multiselectWidget'
     | 'conditionalWidget';
 
-export interface GridLayoutElement {
-    top: number;
-    right: number;
-    bottom: number;
-    left: number;
-}
-
-export interface WidgetPropertiesElement<T> {
-    data: T;
-    addedFrom: 'overview' | 'list';
-    listGridLayout: GridLayoutElement;
-    overviewGridLayout: GridLayoutElement;
-}
-
 export interface WidgetElement<T> {
     id: number;
     key: string;
     widgetId: WidgetType;
+    order: number;
+    section: number;
     title: string;
-    properties: WidgetPropertiesElement<T>;
+    properties: T;
 }
```

Comment:
Are we not using discriminated unions?

Reply:
@tnagorra we don't use this. should we removed unused typings